### PR TITLE
Feature/Fix Whirlpool and revert API retrys

### PR DESCRIPTION
**Pull request by nayr974**
This PR fixes the whirlpool and adds a binary file type definition for .gif files. It also reverts the API retry logic with a hunch that it might be interfering with the metabase fetch.

### CODE REVIEW

File: services/core-web/src/customAxios.js [hunk 1]
```diff
@@ -3,30 +3,20 @@ import { notification } from "antd";
 import { isEmpty } from "lodash";
```
Comment (mmumby) [inline services/core-web/src/customAxios.js]: Was this change intentional? All the deletions to the customAxios?

Comment (nayr974) [reply services/core-web/src/customAxios.js]: Yes, I just updated the description to explain why I updated this file as well :)
